apis/python/src/tiledbsoma/options/_soma_tiledb_context.py

```diff
@@ -234,8 +246,10 @@ def replace(
             if timestamp is _SENTINEL:
                 # Keep the existing timestamp if not overridden.
                 timestamp = self._timestamp_ms
+            if threadpool is None:
+                threadpool = self._threadpool
```

Comment:
Do we want this to be `_SENTINEL` by default, so that you can make this choice?

- leave the existing threadpool (i.e., unset; left as `_SENTINEL`)
- use your own threadpool (i.e., `threadpool=my_threadpool`)
- use a new default threadpool (i.e., `threadpool=None`)

This is a fairly niche use case so I am not _hugely_ concerned if we don’t do this, but just as a thought.

Reply:
I advocate for consistency with the rest of the class. I don't have an opinion which is correct, but if we think the `timestamp` behavior is right, the other params should follow this pattern.

Reply:
Yes, I agree. It will be useful for testing at least. Thanks for suggesting the change.